pkg/kubelet/events/event.go

```diff
@@ -20,6 +20,7 @@ package events
 const (
 	CreatedContainer        = "Created"
 	StartedContainer        = "Started"
+	ContainerDied           = "Died"
```

Comment:
do we have this concept? what is died? crashed or exited or both?

Reply:
It is exited as after that new container will start.